util/gen_log_guide.ts

```diff
@@ -544,7 +544,7 @@ ${structureLog}
 ${translate(language, titles.networkLogLineRegexes)}
 ${regexes.network}
 `;
-        if (regexes.logLine) {
+        if (regexes.logLine !== undefined && regexes.logLine.length > 0) {
```

Comment:
```suggestion
        if (regexes.logLine !== undefined) {
```

This is never empty in the structure above.